.github/workflows/publish-packages.yml

```diff
@@ -225,19 +226,67 @@ jobs:
       - name: Remove artifacts working-directory
         run: rm -rf artifacts
 
-      - name: Update package-locks
+      # Update package-lock files
```

Comment:
What version of npm/node are we running here? Would be nice to update to the latest of each (v8 and v16), if possible.

Reply:
Namely to help avoid lockfile churn

Reply:
Should be running latest by default. We should formalize this though across the board so we don't go back and forth

Reply:
Agreed. We can codify in the engines field in the root package.json.